quarkus/extensions/kogito-quarkus-serverless-workflow-extension/kogito-quarkus-serverless-workflow-deployment/src/main/java/org/kie/kogito/serverless/workflow/parser/schema/OpenApiModelSchemaGenerator.java

```diff
@@ -23,29 +23,35 @@
 import java.util.Map;
 import java.util.Optional;
 
+import org.eclipse.microprofile.config.ConfigProvider;
 import org.eclipse.microprofile.openapi.OASFactory;
+import org.eclipse.microprofile.openapi.models.Components;
 import org.eclipse.microprofile.openapi.models.OpenAPI;
 import org.eclipse.microprofile.openapi.models.Operation;
 import org.eclipse.microprofile.openapi.models.PathItem;
 import org.eclipse.microprofile.openapi.models.media.Content;
 import org.eclipse.microprofile.openapi.models.media.MediaType;
 import org.eclipse.microprofile.openapi.models.media.Schema;
 import org.eclipse.microprofile.openapi.models.media.Schema.SchemaType;
-import org.eclipse.microprofile.openapi.models.tags.Tag;
-import org.jbpm.ruleflow.core.Metadata;
+import org.eclipse.microprofile.openapi.models.responses.APIResponse;
 import org.jbpm.workflow.core.WorkflowModelValidator;
 import org.jbpm.workflow.core.WorkflowProcess;
+import org.kie.kogito.Model;
 import org.kie.kogito.internal.process.runtime.KogitoWorkflowProcess;
 import org.kie.kogito.jackson.utils.ObjectMapperFactory;
 import org.kie.kogito.serverless.workflow.actions.JsonSchemaValidator;
-
-import io.smallrye.openapi.api.util.MergeUtil;
+import org.kie.kogito.serverless.workflow.models.JsonNodeModelInput;
+import org.kie.kogito.serverless.workflow.models.JsonNodeModelOutput;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 public final class OpenApiModelSchemaGenerator {
 
     private OpenApiModelSchemaGenerator() {
     }
 
+    private static final Logger logger = LoggerFactory.getLogger(OpenApiModelSchemaGenerator.class);
+
     private static final Schema ID_SCHEMA;
     private static final String INPUT_SUFFIX = "_input";
     private static final String OUTPUT_SUFFIX = "_output";
```

Comment:
```suggestion
    private static final String INPUT_SUFFIX = "input";
    private static final String OUTPUT_SUFFIX = "output";
```
because the `getSchemaName()` is already putting `_`.

Reply:
Ok, actually what is unneeded is the _ in getSchemaName ;)